### ISSUE TITLE
use storage_path() for Cache.SerializerPath by default

### ISSUE THREAD

**Issue by ceejayoz** [mewebstudio/Purifier#10]
Purifier allows setting `Cache.SerializerPath` to set the location for its serialized cache. This should be set to Laravel's `storage_path()`. For now, it can be set per-config as follows:

```
return array(
    'settings' => array(
        'default' => array(
            'Cache.SerializerPath' => storage_path(),
        ),
    ),
);
```

**Comment by ilumos** [comment]
Yes this is a very sensible idea. The way it is currently requires the serializer path to have `chmod 777` applied otherwise.

**Comment by itbdw** [comment]
permission problem were totally fixed in version 2.0.1 for Laravel 5 version

**Comment by rap2hpoutre** [comment]
@itbdw Does this mean that I have nothing to do and it will work?
